site/content/docs/5.2/examples/_index.md

```diff
@@ -33,8 +33,12 @@ aliases: "/examples/"
                                             alt=""
                                             width="480" height="300"
                                             loading="lazy">
-            <h3 class="h5 mb-1">{{ $example.name }}</h3>
```

Comment:
IDK in terms of a11y but it sounds weird when we navigate with the keyboard (Tab key) because we have to focus the image and then the title each time (and the links are the same). If the user wants to go the last item of the page, it'll double the navigation time.

Reply:
Maybe using `.stretched-link` on existing link would be a better fit?

Reply:
Yep I think `.stretched-link` can be used in this case. While we modify this part, we also need to think about the future where we will probably have an "Edit on StackBlitz" button for each card.